errbot/rendering/__init__.py

```diff
@@ -9,6 +9,12 @@
 # Here are few helpers to simplify the conversion from markdown to various
 # backend formats.
 
+_md_escape_re = re.compile('|'.join(re.escape(c) for c in Markdown.ESCAPED_CHARS))
+
+
+def _md_escape_trans(match):
```

Comment:
Done + the naming convention for the constant was off compared to the other regexp in the file, fixed that too.
thx